raiwidgets/raiwidgets/__version__.py

```diff
@@ -3,4 +3,4 @@
 
 # Version will be populated during release from version.cfg file
 # by running `yarn auto-version`
-version = "0.0.0"
```

Comment:
remove

Reply:
this version is populated by the release script, so make sure to remove this when merging

Reply:
it looks like you need to re-add back the 0.0.0

Reply:
I fixed this for you and pushed to your PR